### DISCUSSION TITLE
How to run multiple Firefox sandbox?

### DISCUSSION THREAD

**Discussion by vukitoso** [netblue30/firejail#4800] [Q&A] [answered]
Hello.
Trying to run a second copy of firefox in a separate sandbox with an empty firefox profile.
But firejail opens a tab in an already running sandbox.
How to run multiple Firefox sandbox?
Thank you.

```
$ firejail --noprofile --private firefox-esr
Parent pid 4432, child pid 4433
Child process initialized in 9.66 ms
Warning: an existing sandbox was detected. /usr/bin/firefox-esr will run without any additional sandboxing features

Parent is shutting down, bye...
```

**Comment by rusty-snake** [comment, marked as answer]
```console
$ firefox --help
[...]
  --no-remote        Do not accept or send remote commands; implies
                     --new-instance.
[...]
```

=> `firejail --private /usr/bin/firefox --no-remote`

**Reply by vukitoso** [reply]
thanks a lot

**Comment by ghost** [comment]
> Warning: an existing sandbox was detected. /usr/bin/firefox-esr will run without any additional sandboxing features

FYI: whenever you see this warning it means you tried starting a program (firefox-esr in this case) with a wrong path. You should use the full path to the program, like /usr/bin/firefox-esr.

We have a [FAQ](https://github.com/netblue30/firejail/wiki/Frequently-Asked-Questions#how-do-i-run-two-instances-of-firefox) on this question.
